root/programs/C#/Frameworks/Infrastructure/Business/RichClient/Business/MyBaseLogic2CS.cs

```diff
@@ -157,7 +157,7 @@ public abstract class MyBaseLogic2CS : BaseLogic2CS
                 else if (parameterValue.ActionType.Split('%')[0] == "ODP")
                 {
                     // Oracle / ODP.NET用のDamを生成
-                    dam = new DamOraOdp();
+                    dam = new DamManagedOdp();
 
                     // 接続文字列をロード（ODP2：Instant Client）
```

Comment:
@daisukenishino 

ODP.NET Managed Driver does not use Oracle Instant Client.
ODP2：Instant Client -> ODP.NET Managed Driver